constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import API from "../utils/api.js";
+import moment from "moment";
+
+export class AFHPage extends React.Component {
+    state = {
+        currentTab: "Math",
+        sessions: [],
+    };
+
+    componentDidMount() {
+        API.get("api/askforhelp/all").then((res) => {
+            const afh = res.data;
+            this.setState({
+                sessions: afh,
+            });
+        });
+    }
+
+    openSubject = (subjectName) => {
+        this.setState({
+            currentTab: subjectName,
+        });
+    };
+
+    render() {
+        let currentSub = this.state.sessions.filter(
+            (session) => session.subject == this.state.currentTab
+        );
+
+        let showSessions = currentSub.map((row, index) => {
+            return <AfhSessionRow key={index} row={row} />;
+        });
+
+        const subjectDisplayNames = {
+            math: "Math",
+            english: "English",
+            computer: "Computer Programming",
+        };
+
+        return (
+            <div id="view-afh">
+                <h1>Ask for Help</h1>
+                <div className="description">
+                    We provide free sessions for students to ask for additional
+                    assistance on any of our program subjects. Please fill the
+                    form to let us know you are coming. You must be registered
+                    with one of our programs to attend.
+                </div>
+
+                <h1>Ask for Help Sessions by Subject</h1>
+                <div className="tab">
+                    <TabButton
+                        currentTab={this.state.currentTab}
+                        openSubject={this.openSubject}
+                        subjectDisplayNames={this.subjectDisplayNames}
+                    />
```

Comment:
I would have 3 `TabButtons` looking like this:
```
<TabButton
          show={this.state.currentSubject == "math"}
          subject={"math"}
 />
<TabButton
          show={this.state.currentSubject == "english"}
          subject={"english"}
 />
<TabButton
          show={this.state.currentSubject == "computer"}
          subject={"computer"}
 />
```

Reply:
The idea here is to use the state variable `currentSubject` (which can be any of lowercase math, english or computer) to decide which tab you want to show.
And given a subject, you can then figure out which display name you want to show.